privacyidea/static/templates/menu.html

```diff
@@ -158,3 +158,11 @@
 <div class="alert alert-warning" ng-show="logoutWarning" translate>
     You will be logged out in {{ myCountdown }} second(s).
 </div>
+<div class="alert alert-warning {{ class_subscription_expired }}"
+    ng-show="subscription_state === 2">
+    <div translate>
+        <p>Your subscription is not valid anymore. Either you exceeded the number of tokens or the
```

Comment:
The text is not centered in the yellow notification.
Please remove the paragraph <p> and it will be centered.

Reply:
done